packages/jsii-rosetta/lib/translate.ts

```diff
@@ -44,7 +44,8 @@ export function translateTypeScript(
  */
 export class Translator {
   private readonly compiler = new TypeScriptCompiler();
-  public readonly diagnostics: ts.Diagnostic[] = [];
+  // eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
```

Comment:
Oh gosh, typescript eslint is not catching up with newer typescript features

Reply:
The culprit appears to be that Babel does not support this just yet 🤷🏻‍♂️ or not in a stable way (couldn't completely understand what I read on the subject)